assets/blocks/RipePineapple.block

```diff
@@ -0,0 +1,8 @@
+{
+  "basedOn": "SimpleFarming:MaturePineapple",
+  "displayName": "Ripe Pineapple",
+  "entity": {
+    "prefab": "RipePineapple"
+  },
+  "tile": "SimpleFarming:PineappleFruit"
```

Comment:
I think the title should be RipePineapple. In this version, the growth stage looks like this:
![terasology-161230185258-1152x720](https://cloud.githubusercontent.com/assets/16467620/21569908/4f9e9be4-cec1-11e6-8dc4-cbf5176f1215.jpg)

What is the PineappleFruit.png for, if not for the last growth stage? :)